pygmt/gridops.py

```diff
@@ -113,3 +113,119 @@ def grdcut(grid, **kwargs):
             result = None  # if user sets an outgrid, return None
 
         return result
+
+@fmt_docstring
+@use_alias(
+           G="outgrid",
+           F="filter",
+           D="distance",
+           I="increment",
+           N="nans",
+           R="region",
+           T="toggle",
+           V="verbose",
+           f="colinfo"
+           )
```

Comment:
If you have `black` installed, you can always run `make format` to automatically format and beautify your codes.

Reply:
I don't, but I will take a look into it. Thanks